test/lib/ufe/testTransform3dTranslate.py

```diff
@@ -239,10 +239,16 @@ def testObservation(self):
         # change to update the bounding box.
         ufe.GlobalSelection.get().append(ball35Item)
 
-        # Move the prim.
+        # Move the prim.  Use a change block to condense USD notifications into
+        # one.  Otherwise, as of 15-Jul-2021, the implementation of
+        # UsdStage::_SetValue() notifies once for creation of the 
+        # 'xformOp:translate' attrSpec, at the end of an SdfChangeBlock, and
+        # once for setting the 'xformOp:translate' field (in Sdf terminology),
+        # again at the end of a separate SdfChangeBlock.
         ball35Prim = usdUtils.getPrimFromSceneItem(ball35Item)
-        ball35Prim.GetAttribute('xformOp:translate').Set(
-            Gf.Vec3d(10, 20, 30))
+        with Sdf.ChangeBlock():
```

Comment:
When I fixed the Transform3d notifications in StagesSubject.cpp to avoid skipping any notifications, this test failed, as it registered two notifications.  Use a change block guard in USD to condense the notifications down to one.